<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1141</width>
      <height>531</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Ajouter formation</string>
     </attribute>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>nom formation</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>code formation</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>nom encadreur</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>270</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>type formation</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>330</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>dure formation</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CODE_FORMATION">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="NOM_FOR">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="NOM_ENCADREUR">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>210</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="TYPE_FOR">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>270</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="DURE_FOR">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>330</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAjouter">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Mod_2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher_button_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>230</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter un code pin </string>
      </property>
     </widget>
     <widget class="QPushButton" name="ImportPDF">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>70</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}
</string>
      </property>
      <property name="text">
       <string>Import PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="insererphoto">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>140</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}</string>
      </property>
      <property name="text">
       <string>inserer photo de formation</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_image">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>230</y>
        <width>181</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Afficher formation</string>
     </attribute>
     <widget class="QTableView" name="tab_for">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>761</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>390</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>code formation</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="code_supp">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>390</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="supp">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>400</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Mod_4">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>360</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher_button">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>440</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qrcodegen">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>260</y>
        <width>200</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Générer QR Code</string>
      </property>
     </widget>
     <widget class="QLabel" name="qrcodecommande">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>60</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>350</y>
        <width>191</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Trier  NOM par ordre:</string>
      </property>
      <widget class="QRadioButton" name="radioButton_croissant">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Alphabétique Croissant</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_dcroissant">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Alphabétique Décroissant</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="rechercher_button_22">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>440</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter un code pin </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Statistique</string>
     </attribute>
     <widget class="QPushButton" name="statsmed">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>430</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Appliquer la statistique</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>notification</string>
     </attribute>
     <widget class="QLineEdit" name="subject">
      <property name="geometry">
       <rect>
        <x>214</x>
        <y>120</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="reset_3">
      <property name="geometry">
       <rect>
        <x>694</x>
        <y>310</y>
        <width>241</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
background-color: rgb(143, 143, 143);
border:2px solid #000;
}
QPushButton {
     color: #fff !important;
text-transform: uppercase;
text-decoration: none;
background: #151B54;

border-radius: 5px;
display: inline-block;
border: none;
transition: all 0.4s ease 0s;
 }
 QPushButton:hover {
     background: #434343;
letter-spacing: 1px;
-webkit-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.57);
-moz-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.57);
box-shadow: 5px 40px -10px rgba(0,0,0,0.57);
transition: all 0.4s ease 0s;
 }</string>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rcpt">
      <property name="geometry">
       <rect>
        <x>214</x>
        <y>70</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TO</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Message</string>
      </property>
     </widget>
     <widget class="QPushButton" name="send_mail">
      <property name="geometry">
       <rect>
        <x>694</x>
        <y>240</y>
        <width>241</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
background-color: rgb(143, 143, 143);
border:2px solid #000;
}
QPushButton {
     color: #fff !important;
text-transform: uppercase;
text-decoration: none;
background: #151B54;

border-radius: 5px;
display: inline-block;
border: none;
transition: all 0.4s ease 0s;
 }
 QPushButton:hover {
     background: #434343;
letter-spacing: 1px;
-webkit-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.57);
-moz-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.57);
box-shadow: 5px 40px -10px rgba(0,0,0,0.57);
transition: all 0.4s ease 0s;
 }</string>
      </property>
      <property name="text">
       <string>SEND</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="msg">
      <property name="geometry">
       <rect>
        <x>224</x>
        <y>170</y>
        <width>393</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
background-color: rgb(255, 255, 255);
border:2px solid #000;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TITLE</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1047</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
